### PR TITLE
LPS-161726 allowing multiple recipients in notification

### PR DESCRIPTION
**Pull request by M-IgorFranca**
Related: https://issues.liferay.com/browse/LPS-161726

### CODE REVIEW

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/source-builder/utils.js [hunk 1]
```diff
@@ -165,8 +165,67 @@ export function parseReassignments(node) {
 	return assignments;
 }
 
-export function parseNotifications(node) {
-	const notifications = {notificationTypes: [], recipients: []};
+function getRecipientsAttibutes(node, xmlDefinition, notificationIndex) {
```
Comment (M-IgorFranca) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/source-builder/utils.js:168]: I believe getReceptionType is a better name for the function as it only returns me attributes of type ReceptionType. But I will wait for the relevant to make possible future changes only once.

**Comment by M-IgorFranca** [comment]
ci:stop

**Comment by M-IgorFranca** [comment]
ci:test:sf

**Comment by M-IgorFranca** [comment]
ci:test:sf

**Comment by M-IgorFranca** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
Just started reviewing :)

**Review by naraandrade — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:test:workflow

**Comment by M-IgorFranca** [comment]
> 

I can see two notification-related tests in this routine:

The[ first test](https://testray.liferay.com/home/-/testray/case_results/1998325618/history?tab=result&hideFilterPin=true) failed, but it has already failed in previous runs.
The [second test ](https://testray.liferay.com/home/-/testray/case_results/1998326363?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D1998325499%26testrayRunId%3D1998325544%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc)passed with success.

**Comment by M-IgorFranca** [comment]
resent [here](https://github.com/brianchandotcom/liferay-portal/pull/124208)

**Comment by naraandrade** [comment]
Just started reviewing :)

**Pull request closed without merging**